include/ceed.h

```diff
@@ -27,19 +27,19 @@
 /// @defgroup CeedOperator CeedOperator: composed FE-type operations on vectors
 ///
 /// @page FunctionCategories libCEED: Types of Functions
-/// @subsection Types of Functions
+/// @section Library Functions
```

Comment:
Should this one by `Library Library Functions`, or drop this level of nesting because it's captured in the @page header? I don't know how this is intended to be used.

![Screenshot_20190911_161919](https://user-images.githubusercontent.com/3303/64739669-f964aa00-d4af-11e9-9a8d-8f9514eebad0.png)

Reply:
I'm not sure what is going on there. I get the same issue when I build master documentation locally. This is a bit of a Doxygen hack to get 'libCEED provides three....' at the top of the page.

Reply:
Does it not work to put it as leading text under `@page`?
http://www.doxygen.nl/manual/commands.html#cmdpage